editor/src/components/canvas/canvas-strategies/reparent-strategy-helpers.ts

```diff
@@ -155,33 +203,16 @@ export function findReparentStrategy(
     strategyState,
   )
 
-  const newParentPath = reparentResult.newParent
```

Comment:
I feel like a juicy conflict with #2457 is coming..

Reply:
*clicks "Squash and merge"*